src/legacy/ui/public/vislib/visualizations/_labels.scss

```diff
@@ -0,0 +1,22 @@
+$visBarLabelDarkColor: $euiColorDarkShade;
+$visBarLabelLightColor: $euiColorLightShade;
+
+.bar-label--normal {
+  font-size: 8pt;
+  pointer-events: none;
+}
+
+.bar-label--stack {
+  font-size: 8pt;
```

Comment:
Don't use `pt` measures here. You can instead use our EUI variable of `$euiSizeS` for `8px`